internal/cmd/gocdk/init.go

```diff
@@ -149,18 +150,26 @@ func inferModulePath(ctx context.Context, pctx *processContext, projectDir strin
 	// so Output can collect stdout.
 	cmd.Stdout = nil
 	gopath, err := cmd.Output()
+	fmt.Println("FROM PCTX-- " + pctx.env[0])
+	fmt.Println("FROM CMD-- " + cmd.Env[0])
+	fmt.Println("WHAT IS HAPPENING HERE-- " + string(gopath))
+	fmt.Println(err)
 	if err != nil {
 		return "", xerrors.Errorf("infer module path: %w", err)
```

Comment:
I'm not 100% sure what is going on in the error you posted, but it seems to be coming from this line and it's reporting that the `go` command exited with a failure status. See if you can pull the stderr from the subprocess and see if that gives you any information.

(My hunch is that because the set of environment variables being given to the `go` command is limited, it might be complaining, but I have no basis for this.)

Reply:
My guess is that it can't find the `go` command because the env does not have a `PATH` in it...?